File: CS/MvcApp_Theme/Models/ThemesModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel;


public enum CommonThemes {
    Aqua,
    DevEx,
    SoftOrange,
    Youthful
}

public class ThemesModel {
    public String Name { get; set; }
    public CommonThemes Theme { get; set; }
}

File: VB/MvcApp_Theme/Models/ThemesModel.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.ComponentModel


Public Enum CommonThemes
    Aqua
    DevEx
    SoftOrange
    Youthful
End Enum

Public Class ThemesModel
    Public Property Name() As String
    Public Property Theme() As CommonThemes
End Class
